docs/writing-tests/testharness-tutorial.md

```diff
@@ -0,0 +1,390 @@
+# testharness.js tutorial
+
+<!--
+Note to maintainers:
+
+This tutorial is designed to be an authentic depiction of the WPT contribution
+experience. It is not intended to be comprehensive; its scope is intentionally
+limited in order to demonstrate authoring a complete test without overwhelming
+the reader with features. This should be updated as typical WPT usage patterns
+change, but please weigh extensions against the demotivating effect that a
+lengthy guide can have on new contributors.
+-->
+
+Let's say you've discovered that WPT doesn't have any tests for how [the Fetch
+API](https://fetch.spec.whatwg.org/) sets cookies from an HTTP response. This
+tutorial will guide you through the process of writing and submitting a test
+for exactly that. Although it includes some very brief instructions on using
+git, you can find more guidance in [the tutorial for git and
+GitHub](../appendix/github-intro).
+
+WPT's testharness.js is a framework designed to help people write tests for the
+web platform's JavaScript APIs. [The testharness.js reference
+page](testharness) describes the framework in the abstract, but for the
```

Comment:
I find the second half of this sentence strange — it implies that there are other ways of testing behaviour, but doesn't really say that, and it doesn't obviously make sense alongside the first half of the sentence. Maybe try rephrasing? Say clearly what you mean.

Reply:
Definitely. Here's what I meant: "we'll only consider the features we need to test the
behavior of `fetch`."

Reply:
Much better